_seattle/API Chaya.md

```diff
@@ -1,18 +1,18 @@
 ---
 name: API Chaya
-address: API Chaya, P.O. Box 14047, Seattle, WA 98114
+address: 
 lat: 
```

Comment:
For a resource without any address, is it appropriate to use the lat/long for Seattle?